content/reference/dos-mitigation.md

```diff
@@ -205,6 +213,8 @@ and `InterceptAccept` for this. For a concrete example, take a look at how Prysm
 implements their [Connection
 Gater](https://github.com/prysmaticlabs/prysm/blob/63a8690140c00ba6e3e4054cac3f38a5107b7fb2/beacon-chain/p2p/connection_gater.go#L43).
 
+js-libp2p has a similar [connection gater](https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#configuring-connection-gater) that can be configured on node start up.
```

Comment:
Should we also callout that js-libp2p provides a config hook for doing rate limiting without needing to do a custom connection gather?  

(Per comments in https://github.com/libp2p/js-libp2p/pull/1421 I do wonder on the utility of providing this ``inboundConnectionThreshold`` because of the effort on documenting, maintaining, and testing vs. just providing the connection gator and letting interested parties handle it like go-libp2p has.)

Reply:
I've linked to the connection rating section, we can always pull it out later if we decide it's not actually useful.